client/src/modules/fill_form/modals/fill_form.modals.html

```diff
@@ -0,0 +1,250 @@
+<form name="FillFormForm" bh-submit="FillFormModalCtrl.submit(FillFormForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.DATA_COLLECTION</li>
+
+      <li class="title">
+        <span translate>TREE.FORMS_CONFIGURATION</span>
+      </li>
+    </ol>
+  </div>
+
+  <div style="background-color: {{ FillFormModalCtrl.dataCollector.color }}; border-radius : 25px ; text-align: center; text-decoration-color: #ffffff; text-shadow: rgb(255, 255, 255) 5px 3px 8px;">
```

Comment:
Can you use [`ngStyle`](https://docs.angularjs.org/api/ng/directive/ngStyle) here instead of hacking in the style?